src/components/AudioPlayer/AudioProgress/songProgress.js

```diff
@@ -0,0 +1,11 @@
+export const convertDurationToProgress = (currentTime, duration) => {
+  const percentage = (currentTime * 100) / duration;
+  return Number.isNaN(percentage) ? 0 : percentage;
```

Comment:
why percentage could be `NaN`? not initialized? looks like the same as previous comments from @tonymtz